### ISSUE TITLE
18134 - Install and configure Ghost with Nginx - Arch Linux

### ISSUE THREAD

**Issue by gnakan** [gnakan/gd-cloud-docs#17]

**Comment by Carla-Johnson** [comment]
Reassigning to me so that I have all 3(?) siblings.

**Comment by Carla-Johnson** [comment]
Please revise, using the edited version of 18132 as a guide.

**Comment by advaiya-main** [comment]
Re-edited as per feedback.

**Comment by Carla-Johnson** [comment]
Discovered that some of my copyedits were removed without explanation. Restored them. Also removed two images (screen-output, welcome screen).

**Comment by advaiya-main** [comment]
Validated testing

**Comment by vishal0soni** [comment]
Tried it again, and still facing the same issue.  Same code that worked earlier is now failing. 
Command failing: cd /var/www/ghost && sudo npm install --production 

Error:
npm ERR! Linux 4.1.6-1-ARCH
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "install" "--production"
npm ERR! node v4.1.1
npm ERR! npm  v3.3.4
npm ERR! code ELIFECYCLE

npm ERR! sqlite3@3.0.8 install: `node-pre-gyp install --fallback-to-build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the sqlite3@3.0.8 install script 'node-pre-gyp install --fallback-to-build'.
npm ERR! This is most likely a problem with the sqlite3 package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-pre-gyp install --fallback-to-build
npm ERR! You can get their info via:
npm ERR!     npm owner ls sqlite3
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /var/www/ghost/npm-debug.log

**Comment by vishal0soni** [comment]
Also tried the steps from https://wiki.archlinux.org/index.php/Ghost 
Still getting the same error for the step "npm install --production" (tried with sudo).
Please suggest.

**Comment by kharise** [comment]
@advaiya-main: Can you revisit this again and send to me for testing on 10/22 after you've verified?

**Comment by advaiya-main** [comment]
I tried it again today, but still getting the same issue. We are not able to get further from the step $cd /var/www/ghost && sudo npm install --production 
Although we got some hints regarding  this command, an article  (https://wiki.archlinux.org/index.php/Ghost) says that there is some way to skip  this step, but when we tried that, that din't worked either.  Need some guidance about how to proceed with this.
